subsys/bluetooth/audio/capabilities.c

```diff
@@ -351,9 +332,18 @@ int bt_audio_capability_register(struct bt_audio_capability *cap)
 	static bool pacs_cb_registered;
 	sys_slist_t *lst;
 
-	if (!cap || !cap->codec) {
-		return -EINVAL;
-	}
+	__ASSERT(cap, "invalid parameters\n");
+	__ASSERT(cap->codec, "invalid parameters\n");
+	__ASSERT(cap->ops, "invalid parameters\n");
+	__ASSERT(cap->ops->config, "invalid parameters\n");
+	__ASSERT(cap->ops->reconfig, "invalid parameters\n");
+	__ASSERT(cap->ops->qos, "invalid parameters\n");
+	__ASSERT(cap->ops->enable, "invalid parameters\n");
+	__ASSERT(cap->ops->start, "invalid parameters\n");
+	__ASSERT(cap->ops->metadata, "invalid parameters\n");
+	__ASSERT(cap->ops->disable, "invalid parameters\n");
+	__ASSERT(cap->ops->stop, "invalid parameters\n");
+	__ASSERT(cap->ops->release, "invalid parameters\n");
```

Comment:
Please be more specific in the string here - Also I think the newline can be omitted, right?

Reply:
Yes you have right, I'll resolve this one as it's not applicable for current PR